sdc/utils.py

```diff
@@ -591,6 +592,20 @@ def sdc_overload(func, jit_options={}, parallel=None, strict=True, inline=None):
     return overload(func, jit_options=jit_options, strict=strict, inline=inline)
 
 
+def sdc_register_jitable(*args, **kwargs):
+    updated_kwargs = kwargs.copy()
+    updated_kwargs['parallel'] = updated_kwargs.get('parallel', config_use_parallel_overloads)
+    updated_kwargs['inline'] = updated_kwargs.get('inline', 'always' if config_inline_overloads else 'never')
```

Comment:
So, does it work?

Reply:
I'm 90% sure it doesn't

Reply:
Yes. If I use e.g. `Always` it prints this:
![image](https://user-images.githubusercontent.com/1541421/73460286-4e368d80-4389-11ea-9553-bc28bf50bce6.png)

Reply:
@AlexanderKalistratov could you propose how to check precisely?

Reply:
The best way is to check IR with enabled/disable option

Reply:
It seems, it doesn't work
https://github.com/numba/numba/issues/5142

Reply:
I have applied proposal patch from the issue.